librz/core/cmd/cmd_regs_meta.inc

```diff
@@ -0,0 +1,124 @@
+// SPDX-FileCopyrightText: 2021 Florian Märkl <info@florianmaerkl.de>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+/**
+ * \file
+ *
+ * Common code for both ar and dr commands
+ * Requires the following defines:
+ *   CMD_REGS_PREFIX     Path inside RzCore to the RzReg *
+ *   CMD_REGS_REG_PATH   Function/Symbol prefix
+ *   CMD_REGS_SYNC       RzCmdRegSync function or NULL
+ */
+
+#define HANDLER__(prefix, name) rz_##prefix##_##name##_handler
+#define HANDLER_(prefix, name)  HANDLER__(prefix, name)
+#define HANDLER(name)           HANDLER_(CMD_REGS_PREFIX, name)
```

Comment:
@ret2libc how about having this accessible everywhere?

Reply:
It is very specific to this file.